src/main/rx/cc2500_frsky_common.h

```diff
@@ -24,13 +24,21 @@
 
 #include "rx/rx_spi.h"
 
+typedef enum {
+  FRSKY_SPI_A1_SOURCE_VBAT = 0,
+  FRSKY_SPI_A1_SOURCE_EXTADC,
+  FRSKY_SPI_A1_SOURCE_CONST
+} frSkySpiA1Source_e;
+
+
 typedef struct rxFrSkySpiConfig_s {
-    uint8_t autoBind;
-    uint8_t bindTxId[2];
-    int8_t  bindOffset;
-    uint8_t bindHopData[50];
-    uint8_t rxNum;
-    uint8_t useExternalAdc;
+    uint8_t               autoBind;
+    uint8_t               bindTxId[2];
+    int8_t                bindOffset;
+    uint8_t               bindHopData[50];
+    uint8_t               rxNum;
+    frSkySpiA1Source_e    a1Source;
```

Comment:
Please do not align - this results in extra change being introduced when longer elements are added (like the case at hand), making the change harder to review.